content/blog/2024/2024-05-15-java-functional-interfaces.md

```diff
@@ -0,0 +1,2028 @@
+---
+authors: [sagaofsilence]
+title: "One Stop Guide to Java Functional Interfaces"
+categories: ["Java"]
+date: 2024-05-01 00:00:00 +1100
+excerpt: "Get familiar with Java Functional Interfaces."
+image: images/stock/0088-jigsaw-1200x628-branded.jpg
+url: one-stop-guide-to-java-functional-interfaces
+---
+
+## Introduction to Functional Programming  
```

Comment:
We could include some information about the history of functional programming, e.g. what programming languages introduced the concept, which other functional languages there are and how they influenced Java.

Reply:
OK. Sure. Will add it.

Reply:
Added